src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -169,9 +165,11 @@ private void processDarCollectionSummariesForResearcher(List<DarCollectionSummar
       //if the latest DAR in the collection has at least one approved dataset,
       //include the create progress report action
       Set<Integer> datasetIds = dataAccessRequestDAO.findDatasetApprovalsByDar(s.getLatestReferenceId());
-      if (!datasetIds.isEmpty()) {
-        s.addAction(DarCollectionActions.CREATE_PROGRESS_REPORT);
-      }
+      // Can only create a progress report if there are approved datasets and no closeout supplement
+      if (!datasetIds.isEmpty() && s.getCloseoutSupplement() == null) {
+          s.addAction(DarCollectionActions.CREATE_PROGRESS_REPORT);
+        }
```

Comment:
This is the actual fix.

Reply:
I'm wondering if this code should also consider the case where closeoutSupplement is not null but the reason list is empty.  Thoughts?

Reply:
From our conversation at standup, @rushtong is going to make a ticket for`consent` input validation so we can keep the current code as-is.